src/Common/Auth/AuthUtils.php

```diff
@@ -517,6 +517,7 @@ public function updatePassword($activeUser, $targetUser, &$currentPwd, &$newPwd,
             $this->errorMessage = xl("Password update error!");
             $this->clearFromMemory($currentPwd);
             $this->clearFromMemory($newPwd);
+            EventAuditLogger::instance()->newEvent('update events', $username, $authGroup, 1, "update password error");
```

Comment:
would use 'password' instead of 'update events' and then be more descriptive in the description. For example, here would do:
'Unsuccessful password change: update password error'

Also, for the unsuccessful logs, the 4th parameter should be a 0 instead of a 1.